### PR TITLE
Creation of doc folder and first wiki page + link from readme

### PR DESCRIPTION
**Pull request by madb1lly**
Hi @fabricedesre ,
As requested, docs folder created and first wiki page added and linked to from the readme.
Cheers :)

### CODE REVIEW

File: docs/Target-devices-and-platforms.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Currently working devices:
```
Comment (fabricedesre) [inline docs/Target-devices-and-platforms.md]: nit: can you rename the file to not use capitals?

File: docs/Target-devices-and-platforms.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Using Gonk-L:
+* Sony Xperia Z3 Compact
+* LG Google Nexus 5
+
+Using Glutin:
+* Linux desktop Glutin
+
+To do: add links to the build instructions for each of the working devices.
+
+# Work in progress
```
Comment (fabricedesre) [inline docs/Target-devices-and-platforms.md]: Not sure we need an empty section here?

File: readme.md [hunk 3]
```diff
@@ -5,7 +5,8 @@ This is an experimental project, aiming at providing a web based environment for
 - Encourage contributions to [Servo](https://servo.org) outside of the mandate of its corporate backers.
 - Scratch an itch and have fun.
 
-It currently runs on Gonk based devices and on desktop platforms (tested on Linux only so far, let me know if you get it to run on Windows or Mac). Another potential target will be the [Librem 5 from Purism](https://puri.sm/shop/librem-5/).
+It currently runs on Gonk based devices and on desktop platforms (tested on Linux only so far, let me know if you get it to run on Windows or Mac). Another potential target will be the [Librem 5 from Purism](https://puri.sm/shop/librem-5/). For more information on target devices and platforms see this page in the [wiki](/docs/Target-devices-and-platforms.md
+).
```
Comment (fabricedesre) [inline readme.md]: It's not a wiki anymore so change to `... see [this page](/docs/target-devices-and-platforms.md).`

File: docs/Target-devices-and-platforms.md (hunk 1 above)
Comment (madb1lly) [reply docs/Target-devices-and-platforms.md]: Yes

File: docs/Target-devices-and-platforms.md (hunk 2 above)
Comment (madb1lly) [reply docs/Target-devices-and-platforms.md]: So if you have something to announce...? ;)

File: readme.md (hunk 3 above)
Comment (madb1lly) [reply readme.md]: Good point. I meant wiki in the general sense, but I agree it's ambiguous on GitHub because there's a separate wiki section.

**Comment by madb1lly** [comment]
Done, I think the pull request has automatically updated.

File: readme.md [hunk 4]
```diff
@@ -5,7 +5,8 @@ This is an experimental project, aiming at providing a web based environment for
 - Encourage contributions to [Servo](https://servo.org) outside of the mandate of its corporate backers.
 - Scratch an itch and have fun.
 
-It currently runs on Gonk based devices and on desktop platforms (tested on Linux only so far, let me know if you get it to run on Windows or Mac). Another potential target will be the [Librem 5 from Purism](https://puri.sm/shop/librem-5/).
+It currently runs on Gonk based devices and on desktop platforms (tested on Linux only so far, let me know if you get it to run on Windows or Mac). Another potential target will be the [Librem 5 from Purism](https://puri.sm/shop/librem-5/). For more information on target devices and platforms see [this page](/docs/Target-devices-and-platforms.md
```
Comment (fabricedesre) [inline readme.md]: the link is broken: s/Target/target

File: docs/target-devices-and-platforms.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+Currently Servonk is targetting Gonk from B2G derived from Android Lollipop. There's also the desktop build with Glutin. 
+
+Using Gonk-L:
+* Sony Xperia Z3 Compact
+* LG Google Nexus 5
+
+Using Glutin:
+* Linux desktop
+
+To do: add links to the build instructions for each of the working devices.
```
Comment (fabricedesre) [inline docs/target-devices-and-platforms.md]: Let's remove that and open an issue instead.

Comment (madb1lly) [reply docs/target-devices-and-platforms.md]: Corrected, pleased excuse me.

File: readme.md (hunk 4 above)
Comment (madb1lly) [reply readme.md]: Corrected, pleased excuse me.

**Pull request merged**